libiocage/lib/errors.py

```diff
@@ -100,14 +100,19 @@ def __init__(self, value, **kwargs):
         )
 
 
-class JailConfigNotFound(Exception):
-    # This is a silent error internally used
-
+class JailConfigNotFound(IocageException):
     def __init__(self, config_type, *args, **kwargs):
         msg = f"Could not read {config_type} config"
+        # This is a silent error internally used
         Exception.__init__(self, msg, *args, **kwargs)
```

Comment:
This isn't correct, `__init__` only accepts `*args` and `**kwargs`. If we ever used this internally, it wouldn't function.

Reply:
There is a problem here - right. But we're seeing different ones here. I would have removed `*args, **kwargs` from Exception.__init__, because we only want to raise this message and nothing else.